api.swagger.json

```diff
@@ -41381,6 +41381,177 @@
+            ]
+          }
+        ]
+      },
+      "post": {
+        "description": "Update State by Key",
+        "operationId": "Update State by Key",
+        "responses": {
+          "200": {
+            "$ref": "#/responses/trait:deprecatable:200"
+          },
+          "400": {
+            "$ref": "#/responses/trait:errorable:400"
+          },
+          "401": {
+            "$ref": "#/responses/trait:errorable:401"
+          },
+          "403": {
+            "$ref": "#/responses/trait:errorable:403"
+          },
+          "404": {
+            "$ref": "#/responses/trait:errorable:404"
+          },
+          "409": {
+            "$ref": "#/responses/trait:conflicting:409"
+          },
+          "500": {
+            "$ref": "#/responses/trait:errorable:500"
+          },
+          "503": {
+            "$ref": "#/responses/trait:errorable:503"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "example": {
+                "version": 1,
+                "actions": [
+                  {
+                    "action": "setName",
+                    "name": {
+                      "en": "New Name"
+                    }
+                  }
+                ]
+              },
+              "$ref": "#/definitions/Update"
+            },
+            "in": "body",
+            "name": "body",
+            "required": true
+          },
+          {
+            "$ref": "#/parameters/trait:expandable:expand"
+          }
+        ],
+        "security": [
+          {
+            "oauth_2_0": [
+              "manage_project:{projectKey}",
+              "manage_orders:{projectKey}"
+            ]
+          }
+        ]
+      },
+      "delete": {
+        "description": "Delete State by Key",
+        "operationId": "Delete State by Key",
+        "responses": {
+          "200": {
+            "$ref": "#/responses/trait:deprecatable:200"
+          },
+          "400": {
+            "$ref": "#/responses/trait:errorable:400"
+          },
+          "401": {
+            "$ref": "#/responses/trait:errorable:401"
+          },
+          "403": {
+            "$ref": "#/responses/trait:errorable:403"
+          },
+          "404": {
+            "$ref": "#/responses/trait:errorable:404"
+          },
+          "409": {
+            "$ref": "#/responses/trait:conflicting:409"
+          },
+          "500": {
+            "$ref": "#/responses/trait:errorable:500"
+          },
+          "503": {
+            "$ref": "#/responses/trait:errorable:503"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/trait:versioned:version"
+          },
+          {
+            "$ref": "#/parameters/trait:expandable:expand"
+          }
+        ],
+        "security": [
+          {
+            "oauth_2_0": [
+              "manage_project:{projectKey}",
+              "manage_orders:{projectKey}"
```

Comment:
the same here you should add "manage_states:{projectKey}", because "manage_orders:{projectKey}" is the deprecated one. See https://docs.commercetools.com/http-api-projects-states#delete-state-by-key